pkg/kritis/metadata/metadata.go

```diff
@@ -71,32 +71,17 @@ type Vulnerability struct {
 // GenericAttestations. It contains signatures, which are verified by
 // public keys held by the AttestationAuthority.
 type RawAttestation struct {
-	AttestationType    AttestationType
-	PgpAttestation     PgpAttestation
-	GenericAttestation GenericAttestation
-}
-
-// PgpAttestation represents the Signature and the Signer Key Id from the
-// containeranalysis Occurrence_Attestation instance.
-type PgpAttestation struct {
-	// OccId is the occurrence ID for containeranalysis Occurrence_Attestation instance
-	OccId     string
-	Signature Signature
-}
-
-// GenericAttestation holds a message payload and Signatures generated over
-// that payload.
-type GenericAttestation struct {
+	SignatureType     SignatureType
+	Signatures        []RawSignature
 	SerializedPayload []byte
-	Signatures        []Signature
 }
 
-// Signature contains the signature content as an opaque bytestring and an ID
-// for the public key that can verify the signature. The ID does not by itself
-// verify the signature. It is merely a key lookup hint.
-type Signature struct {
-	Signature   []byte
+// RawSignature contains the signature content as an opaque bytestring and an
```

Comment:
I got the term bytestring from this definition of a signature: https://github.com/grafeas/grafeas/blob/master/proto/v1beta1/common.proto#L75.

I've updated this comment since we are instead storing the signature as a string.